frontend/src/pages/learn/deciding-when-to-collect-public-pensions.tsx

```diff
@@ -50,20 +80,24 @@ const DecidingWhenToCollectPublicPensions: FC = () => {
         <List disablePadding>
           {[
             {
-              primary: t('key-takeaways.cpp-oas.header'),
-              secondary: t('key-takeaways.cpp-oas.description'),
+              primary: t('key-takeaways.li1'),
+              secondary: t('key-takeaways.li2'),
```

Comment:
Why `li1` and `li2`, etc... It's one list item with a header and description? I prefer how it was structued before with a relevant key name for the entired list item.

Reply:
when you need to maintain locales now and in the future, are you really looking at the key name, or the value of the key?